data/json/overmap/overmap_special/specials.json

```diff
@@ -7697,5 +7697,99 @@
+      { "point": [ 0, 2, -2 ], "overmap": "nursing_home_bb9_north" },
+      { "point": [ 1, 2, -2 ], "overmap": "nursing_home_bb10_north" },
+      { "point": [ 2, 2, -2 ], "overmap": "nursing_home_bb11_north" },
+      { "point": [ 3, 2, -2 ], "overmap": "nursing_home_bb12_north" },
+      { "point": [ 0, 0, -3 ], "overmap": "nursing_home_lab1_north" },
+      { "point": [ 1, 0, -3 ], "overmap": "nursing_home_lab2_north" },
+      { "point": [ 2, 0, -3 ], "overmap": "nursing_home_lab3_north" },
+      { "point": [ 3, 0, -3 ], "overmap": "nursing_home_lab4_north" },
+      { "point": [ 0, 1, -3 ], "overmap": "nursing_home_lab5_north" },
+      { "point": [ 1, 1, -3 ], "overmap": "nursing_home_lab6_north" },
+      { "point": [ 2, 1, -3 ], "overmap": "nursing_home_lab7_north" },
+      { "point": [ 3, 1, -3 ], "overmap": "nursing_home_lab8_north" },
+      { "point": [ 0, 2, -3 ], "overmap": "nursing_home_lab9_north" },
+      { "point": [ 1, 2, -3 ], "overmap": "nursing_home_lab10_north" },
+      { "point": [ 2, 2, -3 ], "overmap": "nursing_home_lab11_north" },
+      { "point": [ 3, 2, -3 ], "overmap": "nursing_home_lab12_north" }
+    ],
+    "connections": [ { "point": [ 3, 3, 0 ], "terrain": "road", "connection": "local_road", "from": [ 3, 2, 0 ] } ],
+    "city_distance": [ 0, 10 ],
+    "occurrences": [ 5, 100 ],
+    "flags": [ "UNIQUE", "RISK_HIGH", "GENERIC_LOOT" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "nursing_home",
+    "locations": [ "land" ],
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "nursing_home_1_north" },
+      { "point": [ 1, 0, 0 ], "overmap": "nursing_home_2_north" },
+      { "point": [ 2, 0, 0 ], "overmap": "nursing_home_3_north" },
+      { "point": [ 3, 0, 0 ], "overmap": "nursing_home_4_north" },
+      { "point": [ 0, 1, 0 ], "overmap": "nursing_home_5_north" },
+      { "point": [ 1, 1, 0 ], "overmap": "nursing_home_6_north" },
+      { "point": [ 2, 1, 0 ], "overmap": "nursing_home_7_north" },
+      { "point": [ 3, 1, 0 ], "overmap": "nursing_home_8_north" },
+      { "point": [ 0, 2, 0 ], "overmap": "nursing_home_9_north" },
+      { "point": [ 1, 2, 0 ], "overmap": "nursing_home_10_north" },
+      { "point": [ 2, 2, 0 ], "overmap": "nursing_home_11_north" },
+      { "point": [ 3, 2, 0 ], "overmap": "nursing_home_12_north" },
+      { "point": [ 0, 0, 1 ], "overmap": "nursing_home_1_roof_north" },
+      { "point": [ 1, 0, 1 ], "overmap": "nursing_home_2_roof_north" },
+      { "point": [ 2, 0, 1 ], "overmap": "nursing_home_3_roof_north" },
+      { "point": [ 0, 1, 1 ], "overmap": "nursing_home_4_roof_north" },
+      { "point": [ 1, 1, 1 ], "overmap": "nursing_home_5_roof_north" },
+      { "point": [ 2, 1, 1 ], "overmap": "nursing_home_6_roof_north" }
+    ],
+    "connections": [ { "point": [ 3, 3, 0 ], "terrain": "road", "connection": "local_road", "from": [ 3, 2, 0 ] } ],
+    "city_distance": [ 0, 5 ],
+    "occurrences": [ 0, 3 ],
```

Comment:
IMO up to three nursing homes per overmap seems excessive. I'd limit it to max 1 per overmap.